### PR TITLE
[build] Fix gbs.conf for base repo @open sesame 2/21 18:01

### PR DESCRIPTION
**Pull request by anyj0527**
- Change latest -> reference to fix CI build failure

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

### CODE REVIEW

File: .TAOS-CI/.gbs.conf [hunk 1]
```diff
@@ -27,7 +27,7 @@ buildroot = ~/GBS-ROOT-Snapshot/
 url = https://api.tizen.org
 
 [repo.base]
-url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/latest/repos/standard/packages/
+url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/reference/repos/standard/packages/
 
 [repo.unified]
 url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/latest/repos/standard/packages/
```
Comment (again4you) [inline .TAOS-CI/.gbs.conf:33]: Please use `reference` instead of `latest`

**Review by again4you — CHANGES_REQUESTED** [review]
Please update `repo.unified` section.

**Review by again4you — APPROVED** [review]
Good Job!

**Review by niley7464 — APPROVED** [review]
LGTM :)

**Review by DonghakPark — APPROVED** [review]
LGTM!

File: .TAOS-CI/.gbs.conf (hunk 1 above)
Comment (anyj0527) [reply .TAOS-CI/.gbs.conf]: Updated!

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**